src/backend/ofdm-processor.cpp

```diff
@@ -242,9 +242,9 @@ void OFDMProcessor::run()
     float       currentStrength;
     int32_t     syncBufferSize  = 32768;
     int32_t     syncBufferMask  = syncBufferSize - 1;
-    float       envBuffer   [syncBufferSize];
```

Comment:
That change is perfectly fine, but there is an alternative available:
If you use 
```c++
constexpr int32_t     syncBufferSize  = 32768;
constexpr int32_t     syncBufferMask  = syncBufferSize - 1;
float envBuffer[syncBufferSize];
```
then `envBuffer` is not a VLA, and it is compliant C++. More details: https://en.cppreference.com/w/cpp/language/array

Reply:
This looks cleaner to me.
@gvanem Please change it.